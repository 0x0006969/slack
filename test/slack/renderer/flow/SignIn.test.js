const { SignIn } = require('../../../../lib/slack/renderer/flow');

describe('SignIn rendering', () => {
  test('works', async () => {
    const message = new SignIn({
      signInLink: 'https://example.org/github/oauth/login?state=abcdef',
    });
    expect(message.toJSON()).toMatchSnapshot();
  });
});
