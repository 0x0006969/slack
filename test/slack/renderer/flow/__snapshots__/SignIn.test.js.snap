// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "#24292f",
      "text": "<https://example.org/github/oauth/login?state=abcdef|Finish connecting your GitHub account>",
    },
  ],
  "response_type": "ephemeral",
}
`;
